faucet/static/index.html

```diff
@@ -0,0 +1,70 @@
+<html lang="en">
+    <head>
+        <meta charset="utf-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
+        <meta name="description" content="" />
+        <meta name="author" content="" />
+        <title></title>
+        <!-- Favicon-->
+        <link rel="icon" type="image/x-icon" href="assets/favicon.ico" />
+        <!-- Core theme CSS (includes Bootstrap)-->
+        <link href="css/styles.css" rel="stylesheet" />
+        <!-- jquery -->
+        <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.6.0/jquery.min.js"></script>
+    </head>
+    <body>
+        <!-- Responsive navbar-->
+        <nav class="navbar navbar-expand-lg navbar-dark bg-dark">
+            <div class="container">
+                <a class="navbar-brand" href="#">🪐 Calibration Faucet</a>
+                <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation"><span class="navbar-toggler-icon"></span></button>
+                <div class="collapse navbar-collapse" id="navbarSupportedContent">
+                    <!-- <ul class="navbar-nav ms-auto mb-2 mb-lg-0">
+                         <li class="nav-item"><a class="nav-link active" aria-current="page" href="#">Home</a></li>
+                         <li class="nav-item"><a class="nav-link" href="#">Link</a></li>
+                         <li class="nav-item dropdown">
+                         <a class="nav-link dropdown-toggle" id="navbarDropdown" href="#" role="button" data-bs-toggle="dropdown" aria-expanded="false">Dropdown</a>
+                         <ul class="dropdown-menu dropdown-menu-end" aria-labelledby="navbarDropdown">
+                         <li><a class="dropdown-item" href="#">Action</a></li>
+                         <li><a class="dropdown-item" href="#">Another action</a></li>
+                         <li><hr class="dropdown-divider" /></li>
+                         <li><a class="dropdown-item" href="#">Something else here</a></li>
+                         </ul> -->
+                        </li>
+                    </ul>
+                </div>
+            </div>
+        </nav>
+        <!-- Page content-->
+        <div class="container">
+            <div class ="center-screen">
+                <div class="text-center align-center mt-5">
+                    <h1>🪐 Calibration Faucet</h1>
+                    <p class="lead">Get some funds and start using the Filecoin Calibration 🚀</p>
+                    <div>
+                        <form id="faucet" action="">
+                            <div class="form-group">
+                                <!-- <label for="addressInput">FIL FaucetAddress</label> -->
+                                <input type="text" class="form-control" name="address" id="address" placeholder="Enter your wallet address (0x....)">
```

Comment:
I would explicitly specify that we only support ETH addresses (unless we end up supporting f4 addresses too). 
```suggestion
                                <input type="text" class="form-control" name="address" id="address" placeholder="Enter your ETH wallet address (0x....)">
```